src/main/java/frc/robot/commands/amp/RetractAmp.java

```diff
@@ -0,0 +1,37 @@
+package frc.robot.commands.amp;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.constants.Constants;
+import frc.robot.subsystems.Amp;
+
+public class RetractAmp extends Command {
+    private Amp amp;
+    private Timer timeout = new Timer();
+
+    public RetractAmp(Amp amp) {
+        this.amp = amp;
+        addRequirements(amp);
+    }
+
+    @Override
+    public void initialize() {
+        timeout.start();
+    }
+
+    @Override
+    public void execute() {
+        amp.setAmpMotorSpeed(Constants.AMP_MOTOR_SPEED);
```

Comment:
This is turning the same direction as "DeployAmp" so one of these is not correct.